lib/image-inspector-client.rb

```diff
@@ -76,10 +78,13 @@ def http_options
     def http_headers
       if @auth_options[:bearer_token]
         {
-          Authorization: "Bearer #{@auth_options[:bearer_token]}"
+          Authorization: "Bearer #{@auth_options[:bearer_token]}",
+          :'X-Auth-Token' => @auth_token
```

Comment:
@ilackarms I think we should also have an option not to send `auth_token` at all if it is `nil`